<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>labelCode</class>
 <widget class="QMainWindow" name="labelCode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1071</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnLogin">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>로그인</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>117</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>종목코드</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCode">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>114</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>종목 코드 입력</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSearch">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>112</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>기본정보 조회</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="pteLog">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>501</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>110</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>기본정보 저장</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSearch2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>당일 체결정보 조회</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSave2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>140</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>당일 체결정보 저장</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGetAllItems">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>전체종목 수집</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSearch_kospi">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>당일체결조회</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="todaySetting">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>yyyy_mm_dd</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
